examples/carplay-web-app-worker/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "carplay-app-worker",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.40",
+    "@types/react": "^18.2.20",
+    "@types/react-dom": "^18.2.7",
+    "buffer": "^6.0.3",
+    "jmuxer": "^2.0.5",
+    "node-carplay": "file:../../",
+    "pcm-player": "^0.0.15",
+    "process": "^0.11.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-loader-spinner": "^5.3.4",
+    "react-scripts": "5.0.1",
+    "stream-browserify": "^3.0.0",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
```

Comment:
@gozmanyoni Only issue I ran into when testing was a type declaration error for web-vitals inside the examples, I bumped the version up to 3.4.0 and that seemed to resolve it

Reply:
Bumped the version and pushed :) 

Glad to hear its working well for you too! 🥳 
As for TS - I thought it will be useful to use here - as the library is quite big, lots of messages, enums and so on - it can become very fragile over time and TS really help in these cases - types are really informative and help when dealing with external lib's or browser api's as well - If you are happy to merge this at any point we can also add github actions that for every branch/PR will verify that:

- Linting rule check passes
- Type Check passes
- Tests Pass